include/eve/arch/riscv/rvv_common_masks.hpp

```diff
@@ -0,0 +1,54 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+namespace eve::detail
+{
+template<arithmetic_scalar_value T, typename N>
+EVE_FORCEINLINE logical<wide<T, N>>
+                rvv_true()
+{
+  static constexpr auto   lmul  = riscv_::getLMUL<T>(N::value);
```

Comment:
there should just be 

```
rvv_lmul_v<T>
```

@jfalcou  - where would you put it.

Reply:
I keep it in tags, as I though tags are used to have some kind of wrapping for real vector registers. Like, it is already have `fundamental_cardinal`, `expected_cardinal`, `is_full` and so on.